### ISSUE TITLE
Add command to exclude member/method from children

### ISSUE THREAD

**Issue by Xriuk** [doxygen/doxygen#8761]
I have some constants which are public, they are defined in the parent class, but it's not really useful to have them visible in children classes, since they are many they take up a lot of space on the screen it would be useful to hide them. Kinda like overriding their visibility to private, to show them only in the parent class.

**Comment by albert-github** [comment]
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.
- Please also specify the **full** doxygen version used (`doxygen -v`).


Cross referencing to https://stackoverflow.com/questions/69062872/doxygen-hide-member-method-from-child-classes

**Comment by Xriuk** [comment]
It's not a bug, it's more a suggestion feature, in this example I wish the static variable `version` (but it's a const most of the times) to be only visible inside Parent, as it is specific to that class.

Attached example:
[Test.zip](https://github.com/doxygen/doxygen/files/7114140/Test.zip)

**Comment by albert-github** [comment]
I don't know the inheritance rules of C# by head so they might be different from the C++.

You use:
```
INLINE_INHERITED_MEMB  = YES
```
implicitly inlining the inherited mebers,

Wouldn't
```
INLINE_INHERITED_MEMB  = NO
```
be already be better:

![image](https://user-images.githubusercontent.com/5223533/132181434-44c7f7b6-f95b-44dd-8337-4105ed12a1e6.png)

**Comment by Xriuk** [comment]
Yeah, this goes against inheritance rules, it would be overriding the default behaviour. I need all the inherited members to be inline, because this approach you mentioned works only for small inheritance relationships, when the tree gets big, the "Additional Inherited Members" gets cluttered badly:
![unknown](https://user-images.githubusercontent.com/6189813/132183960-01b84b94-358b-4246-8c81-70fbced6fa3d.png)

**Comment by albert-github** [comment]
Question from my side, as I couldn't find an answer in the C# standard (ECMA-334, 5th Edition / December 2017)
When I want to access `version` I can access it through `Parent.version` and it will give me the value 2.3.
Can I access the `version` also as `Child.version`?

**Comment by Xriuk** [comment]
Yes you can, it does make little sense for constants though, but it works this way

**Comment by albert-github** [comment]
Looks like after numerous years a new version of the C# Language Definition (Standard ECMA-334 6th Edition i.e. https://www.ecma-international.org/wp-content/uploads/ECMA-334_6th_edition_june_2022.pdf) has been released.
